From 43d59f2c7f1ea791e19d3fb2e0c711543262a34f Mon Sep 17 00:00:00 2001
From: Dave Gerlach <d-gerlach@ti.com>
Date: Fri, 2 Nov 2018 15:59:45 +0530
Subject: [PATCH 18/51] sched / idle: Export cpu_idle_poll_ctrl

commit 04f4ad569c04226e978f8806571b5541207c23e4 from
https://git.ti.com/cgit/processor-sdk/processor-sdk-linux/?h=processor-sdk-linux-01.00.00

Export cpu_idle_poll_ctrl so that it can be used in modules.

Signed-off-by: Dave Gerlach <d-gerlach@ti.com>
Signed-off-by: Xiaolei Wang <xiaolei.wang@windriver.com>
---
 kernel/sched/idle.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kernel/sched/idle.c b/kernel/sched/idle.c
index e4bc4aa739b8..e8ae639d9420 100644
--- a/kernel/sched/idle.c
+++ b/kernel/sched/idle.c
@@ -33,6 +33,7 @@ void cpu_idle_poll_ctrl(bool enable)
 		WARN_ON_ONCE(cpu_idle_force_poll < 0);
 	}
 }
+EXPORT_SYMBOL(cpu_idle_poll_ctrl);
 
 #ifdef CONFIG_GENERIC_IDLE_POLL_SETUP
 static int __init cpu_idle_poll_setup(char *__unused)
-- 
2.17.1

